src/test/java/com/getindata/kafka/connect/iceberg/sink/IcebergSinkSystemTest.java

```diff
@@ -73,8 +66,9 @@ class IcebergSinkSystemTest {
             .withKafkaBootstrap(kafkaContainer.getInternalBootstrap())
             .withPlugin(getJarPath());
 
-    private static final HttpClient httpClient = HttpClient.newHttpClient();;
-    private static SparkTestHelper sparkTestHelper ;
+    private static final HttpClient httpClient = HttpClient.newHttpClient();
+    ;
```

Comment:
Could I ask you to remove additional semicolon?

Reply:
changed in the new commit